### PR TITLE
feat/vue-vite-integration

### PR DESCRIPTION
**Pull request by Ricardo385**
# Pull request type

-   [ ] Bugfix
-   [x] Feature
-   [ ] Code style update (formatting, renaming)
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Build related changes
-   [ ] Documentation content changes
-   [ ] E2E Test(s)
-   [ ] Other (please describe):

# What is the current behavior?
N/A

# What is the new behavior?
Nx Vite integration
Added Vue App
Vue Vite custom configurations created
Added script to run vue app from package json

# Does this introduce a breaking change?

-   [ ] Yes
-   [x] No

# Other information
Please do `npm install`

### CODE REVIEW

File: apps/example-app-vue/project.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
```
Comment (Ricardo385) [inline apps/example-app-vue/project.json:1]: The official Vite Vue app doesn't have a `project.json` so I create a new one with the configs from the Nx official docs --> https://nx.dev/packages/vite/documents/set-up-vite-manually

File: apps/example-app-vue/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+    "extends": "../../tsconfig.base.json",
```
Comment (Ricardo385) [inline apps/example-app-vue/tsconfig.json:2]: added extends to the `tsconfig base` from the root directory

File: apps/example-app-vue/tsconfig.app.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+    "extends": "./tsconfig.json",
```
Comment (Ricardo385) [inline apps/example-app-vue/tsconfig.app.json:2]: added extends to the `tsconfig` from the app. Also this file has been renamed from `tsconfig.node.js` to `tsconfig.app.js` as project structure

File: apps/example-app-vue/project.json (hunk 1 above)
Comment (jedihacks) [reply apps/example-app-vue/project.json:1]: good solution!

File: apps/example-app-vue/tsconfig.json (hunk 2 above)
Comment (jedihacks) [reply apps/example-app-vue/tsconfig.json:2]: Do we need a tsconfig.json and tsconfig.base.json?   I thought eslint or stylelint only use tsconfig.json so it requires both, but I could be wrong

**Review by jedihacks — CHANGES_REQUESTED** [review]
Have some questions on this one but it mostly looks good @Ricardo385

**Pull request merged**